trunk/NDHMS/Routing/Reservoirs/module_read_timeslice_data.F

```diff
@@ -83,13 +90,44 @@ subroutine read_timeslice_data_struct_init(this, start_date, timeslice_path, &
         call read_persistence_netcdf_gage_ids_all(ncid, "gage_id", var_id, this%number_of_reservoir_gages)
 
         allocate (this%reservoir_gage_ids(this%number_of_reservoir_gages))
+        allocate (gage_ids_string_array(this%number_of_reservoir_gages))
 
-        status = nf90_get_var(ncid, var_id, this%reservoir_gage_ids)
+        !status = nf90_get_var(ncid, var_id, this%reservoir_gage_ids)
```

Comment:
Git will track this change; no reason to leave the previous version here

Reply:
Good point. Removing this too.